ethereum/helpers/uris.ts

```diff
@@ -47,27 +47,26 @@ export function getURITemplate(
           'thumb','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_name,
           'thumb_alpha','ipfs://'||renders_cid||'/'||rig_id||'/'||image_thumb_alpha_name,
           'animation_url',animation_base_url||rig_id||'.html',
-          'attributes',json_insert(
-            (
-              select json_group_array(
-                json_object('display_type',display_type,'trait_type',trait_type,'value',value)
-              )
-              from ${attributesTable} where rig_id=ID group by rig_id
-            ),
-            '$[#]',
-            json_object(
-              'display_type','string',
-              'trait_type','Garage Status',
-              'value',coalesce(
-                (select coalesce(end_time, 'in-flight') from ${pilotsTable} where rig_id=ID and end_time is null),
-                'parked'
-              )
-            )
```

Comment:
This was the code that was really hard to further extend. So now there is no more use of `json_insert`.